### PR TITLE
Added Touch event attributes and example with Touch events

### PR DESCRIPTION
**Pull request by aurelijusb**
Touch events as described in:
https://facebook.github.io/react/docs/events.html

### CODE REVIEW

File: project/Build.scala [hunk 1]
```diff
@@ -25,6 +25,7 @@ object ScalajsReact extends Build {
         scalacOptions     ++= Seq("-deprecation", "-unchecked", "-feature",
                                 "-language:postfixOps", "-language:implicitConversions",
                                 "-language:higherKinds", "-language:existentials"),
+        scalacOptions     ++= Seq("-Xmax-classfile-name", "100"),
```
Comment (japgolly) [inline project/Build.scala]: What's this for?

**Comment by japgolly** [comment]
Cooool. :+1: Could you please move the changelog bit down into 0.8.3's section. Only 0.8.2 has been released so far so 0.8.3 is the bucket this patch will go into.

File: project/Build.scala (hunk 1 above)
Comment (aurelijusb) [inline project/Build.scala]: It is written in comit message https://github.com/aurelijusb/scalajs-react/commit/ecbd83b3d4f44bf6f83eb58d670dad7ef0b6917a

On my Ubuntu machine I have encrypted home folder.
During compilation Scala creates very long class names and I get
"File name too long" exception.
So this is a small hack to make compilation work on encrypted file system.

**Comment by aurelijusb** [comment]
`changelog` updated.

File: project/Build.scala (hunk 1 above)
Comment (japgolly) [inline project/Build.scala]: Ah thanks.

Ok so have a read of this: http://scala-language.1934581.n4.nabble.com/Xmax-classfile-name-td4637574.html

This option seems quite dangerous. It affects binary compatibility and needs all dependencies to have the same setting, else :boom:.

Let's remove this and get the rest of the patch in.

**Comment by aurelijusb** [comment]
Reverted `Xmax-classfile-name` commit. Now only TouchEvent chagnes are left.

**Comment by japgolly** [comment]
Awesome, thanks for the patch.

**Pull request merged**